monitoring-bot/internal/monitor.go

```diff
@@ -194,44 +198,60 @@ func (m *Monitor) Start() {
 		if err != nil {
 			log.Error().Err(err).Msg("monitoring networks failed with error")
 		}
+
+		log.Debug().Msg("monitoring stellar bridges")
+		if err := m.monitorBridges(); err != nil {
+			log.Error().Err(err).Msg("monitoring bridges failed")
+		}
 	}
 }
 
 // getTelegramUrl returns the telegram bot api url
 func (m *Monitor) getTelegramURL() string {
-	return fmt.Sprintf("https://api.telegram.org/bot%s", m.env.botToken)
+	return fmt.Sprintf("%s%s", telegramBotURL, m.env.botToken)
 }
 
 // sendMessage sends a message with the balance to a telegram bot
 // if it is less than the tft threshold
 func (m *Monitor) sendMessage(manager client.Manager, wallet wallet) error {
-	balance, err := m.getBalance(manager, wallet.Address)
+	con, err := manager.Substrate()
+	if err != nil {
+		return err
+	}
+	defer con.Close()
+
+	balance, err := m.getBalance(con, wallet.Address)
 	if err != nil {
 		return err
 	}
 
 	if balance >= float64(wallet.Threshold) {
 		return nil
 	}
+	return m.sendBotMessage(fmt.Sprintf("wallet %v with address:\n%v\nhas balance = %v ⚠️", wallet.Name, wallet.Address, balance))
+}
 
+func (m *Monitor) sendBotMessage(msg string) error {
 	url := fmt.Sprintf("%s/sendMessage", m.getTelegramURL())
 	body, _ := json.Marshal(map[string]string{
 		"chat_id": m.env.chatID,
-		"text":    fmt.Sprintf("wallet %v with address:\n%v\nhas balance = %v ⚠️", wallet.Name, wallet.Address, balance),
+		"text":    msg,
 	})
-	response, err := http.Post(
+
+	res, err := http.Post(
 		url,
 		"application/json",
 		bytes.NewBuffer(body),
 	)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to send http post request: %w", err)
```

Comment:
can we make error msgs -> `failed to send telegram message request` 
- for res.StatusCode >= 400 also